Classes/Settings/SettingsViewController.swift

```diff
@@ -259,3 +268,69 @@ NewIssueTableViewControllerDelegate {
         showDetailViewController(navigation, sender: nil)
     }
 }
+
+extension SettingsViewController {
+  // Default Reaction + MenuController
+  
+   private func showDefaultReactionMenu() {
+    
+    setDefaultReaction.becomeFirstResponder() // Required
+    
+    let actions = [
+      (ReactionContent.thumbsUp.emoji, #selector(SettingsViewController.onThumbsUp)),
+      (ReactionContent.thumbsDown.emoji, #selector(SettingsViewController.onThumbsDown)),
+      (ReactionContent.laugh.emoji, #selector(SettingsViewController.onLaugh)),
+      (ReactionContent.hooray.emoji, #selector(SettingsViewController.onHooray)),
+      (ReactionContent.confused.emoji, #selector(SettingsViewController.onConfused)),
+      (ReactionContent.heart.emoji, #selector(SettingsViewController.onHeart))
+    ]
+    
+    let menu = UIMenuController.shared
+    menu.menuItems = actions.map { UIMenuItem(title: $0.0, action: $0.1) }
+    menu.setTargetRect(defaultReactionLabel.frame, in: setDefaultReaction)
+    menu.setMenuVisible(true, animated: trueUnlessReduceMotionEnabled)
+  }
+  
+  override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
+    switch action {
+    case #selector(SettingsViewController.onThumbsUp),
+         #selector(SettingsViewController.onThumbsDown),
+         #selector(SettingsViewController.onLaugh),
+         #selector(SettingsViewController.onHooray),
+         #selector(SettingsViewController.onConfused),
+         #selector(SettingsViewController.onHeart):
+      return true
+    default: return false
+    }
+  }
+  
+  @objc private func onThumbsUp() {
+    updateDefaultReaction(.thumbsUp)
+  }
+  
+  @objc private func onThumbsDown() {
+    updateDefaultReaction(.thumbsDown)
+  }
+  
+  @objc private func onLaugh() {
+    updateDefaultReaction( .laugh)
+  }
+  
+  @objc private func onHooray() {
+    updateDefaultReaction(.hooray)
+  }
+  
+  @objc private func onConfused() {
+    updateDefaultReaction(.confused)
+  }
+  
+  @objc private func onHeart() {
+   updateDefaultReaction(.heart)
+  }
+  
+  func updateDefaultReaction(_ reaction: ReactionContent)
+  {
+    UserDefaults.setDefault(reaction: reaction)
+    defaultReactionLabel.text = reaction.emoji
+  }
+}
```

Comment:
Fine merging this as-is, but if we're going to put stuff in an `extension`, we might as well put it in another file. We don't really follow the trend of making extensions for everything in this project.

Reply:
I'll just remove the extension no biggy.